Modules/System/WebDAV/src/Model/WebDAVContent.Table.al

```diff
@@ -0,0 +1,66 @@
+table 5678 "WebDAV Content"
+{
+    Access = Public;
+    DataClassification = SystemMetadata; // Data classification is SystemMetadata as the table is temporary
+    Caption = 'WebDAV Folder';
+    TableType = Temporary;
+    Extensible = false;
+
+    fields
+    {
+        field(1; "Entry No."; Integer)
+        {
+            Caption = 'Entry No.';
+            AutoIncrement = true;
+        }
+        field(10; Name; Text[250])
+        {
+            Caption = 'Title';
+        }
+        field(11; "Full Url"; Text[2048])
+        {
+            Caption = 'Full Url';
+        }
+        field(12; "Relative Url"; Text[2048])
+        {
+            Caption = 'Relative Url';
+        }
+        field(13; Level; Integer)
+        {
+            Caption = 'Level';
+        }
+        field(20; "Is Collection"; Boolean)
```

Comment:
This jumps from 13 to 20. Again, there might be a reason, but would it not be better just to keep incrementing by 1?

Reply:
fields are grouped by context and the space between groups makes it easier to add fields later, see older NAV objects

Reply:
I agree that it happens with older NAV objects, but not sure that is a reason to continue the practice. I will defer to Microsoft's opinion on the matter though.

Reply:
I don't have a strong opinion on this as long as it makes sense. Internally, unless it's a localized (US, DK etc) field, we've been going sequentially for new tables in general.
I do notice it jumps from 20 to 22. Any reason 21 is missing?